File: Cursos/MasterLogicaProgramacion/EjerciciosPython/Ej10.py
def escalera(escalones):
    retorno = ""
    for i in range(1, (escalones + 1)):
        retorno += generarEscalon(i)

    return retorno

def generarEscalon(cant):
    return ("[-]" * cant) + "\n"

print(escalera(4))

File: Cursos/MasterLogicaProgramacion/EjerciciosJavascript/Ej10.js
function escalera(escalones) {
    let retorno = "";
    for (let i = 1; i <= escalones; i++) {
        retorno += generarEscalon(i);
    }

    return retorno;
}

function generarEscalon(cant){
    return "[-]".repeat(cant) + "\n";
}

console.log(escalera(4));
